### PR TITLE
Milestone: making a Popup window

### PR DESCRIPTION
**Pull request by fasaldana**
In this milestone, we created a pop-up modal to show more about a project at the work section:
- Added functionality on a JavaScript file.
- Added Styles to the modal window

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,8 +168,38 @@ <h2 class="box-title">Professional Art Printing Data</h2>
...
+                        <ul>
+                            <li>html</li>
+                            <li>Boostrap</li>
+                            <li>Ruby on rails</li>
+                        </ul>
+                    </div>
+                    <div class="modal-content">
+                        <img class="modal-img" src="media/work_example.png" alt="work image">
+                        <div class="modal-about">
+                            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
```
Comment (elyor-doniyorov) [inline index.html:188]: - [ ] I recommend removing commented code because it makes your codebase unclean.

File: works.css [hunk 2]
```diff
@@ -42,6 +42,8 @@ section.works {
   margin-top: 80px;
   width: 28rem;
   align-self: center;
+  padding-left: 30px;
+  margin-left: 20px;
 }
```
Comment (elyor-doniyorov) [inline works.css:48]: - [ ] In your mobile popup there is a space under buttons that differs from the given design. Take a look at the screenshot below to see the difference 👇 
- Figma design 👇 
Screenshot (223)

- Your design with space 👇 
Screenshot (222)

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @fasaldana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: work-cards.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  {
+    name: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    featuredImage: 'media/work_example.png',
+    tecnologies: ['Html', 'Bootstrap', 'Ruby on rails'],
+    liveVer: 'https://fasaldana.github.io/Portfolio_project/',
+    source: 'https://github.com/fasaldana/Portfolio_project',
+  },
+];
+
```
Comment (BenMukebo) [inline work-cards.js:12]: - [ REQUIRED] Please consider adjusting the padding-left on this section, you have applied the overflow property, which was right but the are some issues that are still not addressed yet, see this image below, your cards are not centered properly, please review it 
- 
marting

- [ OPTIONAL] please consider taking a look back at the mobile version too, see how it behave now, I would strongly recommend you adjust it too.
-  
tiiu

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### Hi ** @fasaldana, **,

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**